templates/web/fixamingata/about/faq-sv.html

```diff
@@ -1,7 +1,5 @@
 [% INCLUDE 'header.html', title => loc('Frequently Asked Questions'), bodyclass => 'twothirdswidthpage' %]
 
-[% INCLUDE 'about/_sticky-button-content-navbar.html' %]
```

Comment:
This makes the navigation easily navigable on mobile, see www.fixmystreet.com/faq - is there a reason you removed it, or can we leave it in?

Reply:
Please leave it in, thank you!